SmalltalkSources/SBE-Testing.package/SBEScriptBuilderTest.class/instance/testInform.st

```diff
@@ -1,6 +1,9 @@
 testing
 testInform
 
+	self fail: 'ct: Not yet implemented.'.
+	self flag: #discuss. "Not sure whether we want to block this at all 🤔"
```

Comment:
Is this already relevant for merging the current feature or is this only relevant for future ones?

Reply:
Only relevant for future ones. This would notify us of broken figure scripts earlier, at the moment, the CI would just timeout.